.github/workflows/copilot_labeller.yml

```
@@ -0,0 +1,149 @@
+                  id
+                }
+              }
+            }' > discussion_data.json
+
+          echo 'DISCUSSION_BODY_HTML='$(jq -r '.data.repository.discussion.bodyHTML' discussion_data.json) >> $GITHUB_ENV
+          echo 'DISCUSSION_ID='$(jq -r '.data.repository.discussion.id' discussion_data.json) >> $GITHUB_ENV
+      - run: npm install jsdom
+      
+      - name: Get selected Copilot feature area
+        id: get_selected_feature_area
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)